packages/transaction_signing_gateway/lib/gateway/transaction_signing_gateway.dart

```diff
@@ -33,6 +38,12 @@ class TransactionSigningGateway {
         password: password,
       );
 
+  /// Signs the passed [transaction].
+  ///
+  /// This will trigger the entire signing flow, where first a transaction summary is shown to the user,
+  /// if accepted it then looks for a capable [TransactionSigner].
+  /// After successful credentials retrieval and transaction signing, a [SignedTransaction] object is spit out.
+  /// If any of the steps fail, a [TransactionSigningFailure] is returned.
```

Comment:
```suggestion
  /// This transaction triggers the entire signing flow, where a transaction summary is first shown to the user. 
  /// If the transaction is accepted, the workflow looks for a capable [TransactionSigner].
  /// After a credentials retrieval and transaction signing is successful, a [SignedTransaction] object is output.
  /// If any of the steps fail, a [TransactionSigningFailure] is returned.
```

Reply:
one of our favorite grammar habits (all of us!) is the [unclear reference](https://developers.google.com/style/word-list#letter-t). 

> this, that

> Where possible, put a noun after _this_ or _that_ for clarity. If doing so results in clunky prose, then don't do it; but even then, try thinking about what the noun would be. If you aren't sure what noun this or that refers to, then consider rephrasing.

I think we mean "transaction" in

> This will trigger the entire signing flow
> If accepted

I really like improving our user experience by simplifying the way we use language. Tip: see our [Tendermint technical content style guidance](https://www.notion.so/allinbits/Technical-content-style-guidance-b896877847a547518d3dff33fac7366a) page for all sorts of grammar goodness.